packages/tools/tests/test/audioV2/shared/abstractAudioNode.volume.ts

```diff
@@ -99,15 +99,17 @@ export const AddSharedAbstractAudioNodeVolumeTests = (audioNodeType: AudioNodeTy
         test.describe("Default ramp", () => {
             test("Ramping volume from 0 to 1 over 1 second should play sound at 0.1x volume at 0.1 seconds with default linear shape", async ({ page }) => {
                 await EvaluateAbstractAudioNodeTestAsync(page, audioNodeType, async ({ audioNodeType }) => {
-                    await AudioV2Test.CreateAudioEngineAsync(audioNodeType, undefined, { volume: audioNodeType === "AudioEngineV2" ? 0 : 1 });
+                    const audioEngine = await AudioV2Test.CreateAudioEngineAsync(audioNodeType, undefined, { volume: audioNodeType === "AudioEngineV2" ? 0 : 1 });
                     const { sound, outputNode } = await AudioV2Test.CreateAbstractSoundAndOutputNodeAsync(audioNodeType, audioTestConfig.pulseTrainSoundFile, {
                         volume: audioNodeType !== "AudioEngineV2" ? 0 : 1,
                     });
 
-                    outputNode.setVolume(1, { duration: 1 });
-                    sound.play();
-                    await AudioV2Test.WaitAsync(1, () => {
-                        sound.stop();
+                    await AudioV2Test.WaitAsync(audioContext instanceof OfflineAudioContext ? audioEngine.parameterRampDuration : 0, async () => {
```

Comment:
this pattern has been changed in a few places it seems, could it be factorized ?

Reply:
Addressed by commit https://github.com/BabylonJS/Babylon.js/pull/16839/commits/e5cc9a13fa75ebf9eb24588023aee642f1f33411